composer.json

```diff
@@ -3,16 +3,16 @@
     "description": "Bundle for creating forms in Sulu.",
     "type": "sulu-bundle",
     "require": {
-        "symfony/symfony": "^2.8.7 || ^3.0",
-        "symfony/swiftmailer-bundle": "^2.3 || ^3.0",
+        "symfony/symfony": "^2.8.7 || ^3.0 || ^4.0",
+        "symfony/swiftmailer-bundle": "^2.3 || ^3.0 || ^4.0",
```

Comment:
it seems the swiftmailer is versionated not the same way as symfony so we need remove ^4.0

```
        "symfony/swiftmailer-bundle": "^2.3 || ^3.0",
```